text/0000-canonical-metadata.md

```diff
@@ -1216,6 +1216,172 @@ interface StaticResourceReference {
 ```
 </details>
 
+## Metadata Shape Option 2
+Overview:
+The vision is to allow expert users to retrieve file based metadata and for the regular users to retrieve an easy, self-contained metadata object that exposes all publicly available/accessible properties of the current bundle.
+
+The main differences in proposed shape #2:
+- no decorators: external consumer shouldn't be concerned if the class fields are decorated or not. All they need is the information that describes the component's/module's public interface. With that, instead of collecting decorators, each class member will carry 'isPublic' property to indicate its access. 
```

Comment:
```suggestion
- no decorators: external consumer shouldn't be concerned if the class fields are decorated or not. All they need is the information that describes whether the class member is public. With that, instead of collecting decorators, each class member will carry 'isPublic' property to indicate its access. 
```
```suggestion
- no decorators: external consumer shouldn't be concerned if the class fields are decorated or not. All they need is the information that describes the component's/module's public interface. With that, instead of collecting decorators, each class member will carry 'isPublic' property to indicate its access. 
```
```suggestion
- no decorators: external consumer shouldn't be concerned if the class fields are decorated or not. All they need is the information that describes whether the class member is public. With that, instead of collecting decorators, each class member will carry 'isPublic' property to indicate its access. 
```